code/game/gamemodes/objective.dm

```diff
@@ -876,7 +885,34 @@ GLOBAL_LIST_EMPTY(possible_items_special)
 					stolen_count++ //it counts. nice.
 			else if(is_type_in_typecache(I, wanted_items))
 				stolen_count++
-	return stolen_count >= 5
+	return stolen_count >= amount
+
+/datum/objective/steal_n_of_type/organs
+	name = "steal organs"
+	explanation_text = "Steal at least 10 organs! They must be kept healthy."
```

Comment:
Should be [amount] organs not 10
The code says 5 organs, and you should specify live, non-robotic organs

Reply:
just fyi, this would update in game to be accurate but yeah i should make the code the same